www/codePushUtil.ts

```diff
@@ -66,7 +66,11 @@ class CodePushUtil {
      */
     public static logError(message: String, error?: Error): void {
         var errorMessage = message || "" + " " + CodePushUtil.getErrorMessage(error);
-        console.error(CodePushUtil.TAG + " " + errorMessage);
+        var stackTrace = '';
```

Comment:
It would be great to use `let` instead of `var` and `const` instead of `let` if possible.

Reply:
Done, thx